src/transform.ts

```diff
@@ -47,10 +47,28 @@ export const transform = (
 ): OutputNode => {
   const type = getNgType(node);
   switch (type) {
+    case 'Unary': {
+      // @ts-ignore
+      const { left, right, expr } = node as ng.Unary;
+      const isPrefixMinus = right === expr;
+      const tArgument = _t<b.Expression>(isPrefixMinus ? right : left);
+      return _c<b.UnaryExpression>(
+        'UnaryExpression',
+        {
+          prefix: true,
+          argument: tArgument,
+          operator: isPrefixMinus ? '-' : '+',
+        },
+        node.span,
+        { hasParentParens: isInParentParens },
+      );
```

Comment:
`left` and `right` seem to be only used for backward compatibility, and it seem to be not recommended to use as they typed them as `never` in their type, which we should probably avoid using them here:

```suggestion
      // @ts-ignore: there is no `Unary` in `@angular/compiler@<10.1.0` 
      const { operator, expr } = node as ng.Unary;
      const tArgument = _t<b.Expression>(expr);
      return _c<b.UnaryExpression>(
        'UnaryExpression',
        {
          prefix: true,
          argument: tArgument,
          operator: operator as '-' | '+',
        },
        node.span,
        { hasParentParens: isInParentParens },
      );
```